code/modules/cargo/packs.dm

```diff
@@ -417,6 +417,13 @@
 	contains = list(/obj/item/weapon/storage/box/exileimp)
 	crate_name = "exile implant crate"
 
+/datum/supply_pack/security/armory/ionrifles
+	name = "Ion Carbines Crate"
+	cost = 2500
```

Comment:
I suggest 8000 points as cost
I suggest renaming it to ion rifle crate
I suggest adding 1 rifle

Heres your balance